agents/scripts/install-dependencies.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+set -e
+
+scripts_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+set -a
+source "$scripts_dir/config.env"
+source "$scripts_dir/${ENV_FILE}"
+set +a
+
+echo "[INFO] Updating and installing system packages..."
+apt-get clean && rm -rf /var/lib/apt/lists/* && apt-get update
+apt-get install -y build-essential zlib1g-dev wget git
+
```

Comment:
Set default value if ENV_FILE not found. 
```
if [ ! -f "$scripts_dir/${ENV_FILE}" ]; then
    echo "[WARN] ENV file not found: $scripts_dir/${ENV_FILE}, skipping."
else
    source "$scripts_dir/${ENV_FILE}"
fi
```

Reply:
Added safeguards for sourcing `config.env` and `ENV_FILE` + added default values for env variables if they aren't set.